src/__tests__/components/introspection/GetFieldWrapper.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { mockGqlSchemaIntrospection } from '@/__tests__/__mocks__/mockGqlSchemaIntrospection';
+import { GetFieldWrapper } from '@/app/components/introspection/shared/GetFieldWrapper';
+import { IIntrospectionSchema } from '@/types/introspectionImportedTypes';
+
+import { render } from '@testing-library/react';
+
+describe('GetFieldWrapper', () => {
+  it('should not render component if kind not present', () => {
+    const { container } = render(
+      <GetFieldWrapper
+        kind={undefined as unknown as string}
+        name={'name'}
+        schema={{} as unknown as IIntrospectionSchema}
+      />
+    );
+
+    expect(container.firstChild).toBeNull();
+  });
+
+  it('should not render component if name not present', () => {
+    const { container } = render(
+      <GetFieldWrapper
+        kind={'OBJECT'}
+        name={undefined as unknown as string}
+        schema={{} as unknown as IIntrospectionSchema}
```

Comment:
The problem is, that this is part of Documentation Explorer. For gql implementation that essentially means, we are asking gql endpoint to tell us about it's schema.

We assume, that returned object follows interfaces, as it should, but we actually don't know. 
Also we cannot predict the exact structure of returned object on gql introspection.

That is why we basically are guessing and checking.

This component have to receive name and schema to be used, but this, because uncertainties in received json, could not happen in runtime.

That is why I would prefer not to mention this in component's prop-type, but handle this situation in component and tests